doc/modules/classes.rst

```diff
@@ -1360,23 +1360,6 @@ Low-level methods
 Recently deprecated
 ===================
 
-To be removed in 0.19
```

Comment:
I mean "should 0.19 list everything that was removed". It might be useful but maybe not necessarily part of this PR ;) Do I have your +1?